src/components/dialogs/parameters/voltageinit/voltage-limits-parameters.js

```diff
@@ -78,28 +74,107 @@ const VoltageLimitsParameters = () => {
+        return [
+            {
+                label: 'VoltageLevelFilter',
+                dataKey: FILTERS,
+                initialValue: [],
+                editable: true,
+                directoryItems: true,
+                equipmentTypes: [EQUIPMENT_TYPES.VOLTAGE_LEVEL],
+                elementType: elementType.FILTER,
+                titleId: 'FiltersListsSelection',
+                extra: VoltageLevelFilterTooltip,
+            },
+            {
+                label: 'LowVoltageLimitDefault',
+                dataKey: LOW_VOLTAGE_LIMIT,
+                initialValue: null,
+                editable: true,
+                numeric: true,
+                adornment: VoltageAdornment,
+                textAlign: 'right',
+            },
+            {
+                label: 'HighVoltageLimitDefault',
+                dataKey: HIGH_VOLTAGE_LIMIT,
+                initialValue: null,
+                editable: true,
+                numeric: true,
+                adornment: VoltageAdornment,
+                textAlign: 'right',
+            },
+        ].map((column) => ({
+            ...column,
+            label: intl.formatMessage({ id: column.label }).toUpperCase(),
+        }));
+    }, [VoltageLevelFilterTooltip, intl]);
+
+    const newModificationRowData = useMemo(() => {
         const newRowData = {};
         newRowData[SELECTED] = false;
-        COLUMNS_DEFINITIONS.forEach(
+        VOLTAGE_LIMITS_MODIFICATION_COLUMNS_DEFINITIONS.forEach(
             (column) => (newRowData[column.dataKey] = column.initialValue)
         );
         return newRowData;
-    }, [COLUMNS_DEFINITIONS]);
-    const createVoltageLimitRows = () => [newRowData];
+    }, [VOLTAGE_LIMITS_MODIFICATION_COLUMNS_DEFINITIONS]);
+
+    const createVoltageLimitModificationRows = () => [newModificationRowData];
+
+    const newDefaultRowData = useMemo(() => {
+        const newRowData = {};
+        newRowData[SELECTED] = false;
+        VOLTAGE_LIMITS_DEFAULT_COLUMNS_DEFINITIONS.forEach(
+            (column) => (newRowData[column.dataKey] = column.initialValue)
+        );
+        return newRowData;
+    }, [VOLTAGE_LIMITS_DEFAULT_COLUMNS_DEFINITIONS]);
+
+    const createVoltageLimitDefaultRows = () => [newDefaultRowData];
+
+    const useVoltageLimitsModificationFieldArrayOutput = useFieldArray({
+        name: `${VOLTAGE_LIMITS_MODIFICATION}`,
+    });
+
+    const useVoltageLimitsDefaultFieldArrayOutput = useFieldArray({
+        name: `${VOLTAGE_LIMITS_DEFAULT}`,
+    });
 
     return (
-        <DndTable
-            arrayFormName={`${VOLTAGE_LIMITS}`}
-            columnsDefinition={COLUMNS_DEFINITIONS}
-            useFieldArrayOutput={useFieldArrayOutput}
-            createRows={createVoltageLimitRows}
-            tableHeight={270}
-            withAddRowsDialog={false}
-            withLeftButtons={false}
-        />
+        <>
+            <Typography component="span" variant="h6">
```

Comment:
Why not having a DialogContent here like in `EquipmentSelectionParameters` and a Grid Layout with some titles like other places in parameters ? I mean it's the only reference to a Typography h6 in the code.

Reply:
I'll try it out

Reply:
I've replaced Typography with some Grid component combined with css